internal/utils/telegraf/accumulator/accumulator.go

```diff
@@ -0,0 +1,58 @@
+package accumulator
+
+import (
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/signalfx-agent/internal/utils/telegraf/emitter"
+)
+
+// Accumulator is an interface used to accumulate telegraf measurements from
+// Telegraf plugins.
+type Accumulator struct {
+	Emitter emitter.Emitter
```

Comment:
This may not need to be exported if it isn't meant to be used outside of the methods on this class.